src/lib.rs

```diff
@@ -11,30 +11,53 @@ type EnvMap = HashMap<String, String>;
 
 pub struct Vcvars {
     env_map: Option<EnvMap>,
+    prerelease: bool,
 }
 
 impl Vcvars {
-    //! Runs vcvars in a `cmd.exe` child process (at most once) and makes available the set of environment variables the child process inherited, mutated by vcvars. The `cmd.exe` stdout output is converted with [`std::string::String::from_utf8_lossy()`].
-    //!
-    //! Use [`std::env::split_paths()`] to split a variable like `INCLUDE`, which could then, e.g., be passed to [`cc::Build::includes()`].
-    //!
-    //! # Example
-    //!
-    //! ```ignore
-    //! let mut vcvars = Vcvars::new();
-    //! let vcvars_include = vcvars.get_cached("INCLUDE").unwrap();
-    //!
-    //! cxx_build::bridge("src/demo.rs")
-    //!     .file("src/demo.cc")
-    //!     .includes(env::split_paths(&*vcvars_include))
-    //!     .compile("demo");
-    //! ```
-
     pub fn new() -> Self {
         #![must_use]
         #![allow(clippy::new_without_default)]
+        //! Runs vcvars in a `cmd.exe` child process (at most once) and makes available the set of environment variables the child process inherited, mutated by vcvars. The `cmd.exe` stdout output is converted with [`std::string::String::from_utf8_lossy()`].
+        //!
+        //! Use [`std::env::split_paths()`] to split a variable like `INCLUDE`, which could then, e.g., be passed to [`cc::Build::includes()`].
+        //!
+        //! # Example
+        //!
+        //! ```ignore
+        //! let mut vcvars = Vcvars::new();
+        //! let vcvars_include = vcvars.get("INCLUDE").unwrap();
+        //!
+        //! cxx_build::bridge("src/demo.rs")
+        //!     .file("src/demo.cc")
+        //!     .includes(env::split_paths(&*vcvars_include))
+        //!     .compile("demo");
+        //! ```
+
+        Self {
+            env_map: None,
+            prerelease: false,
+        }
+    }
 
-        Self { env_map: None }
+    pub fn use_vs_prerelease(mut self) -> Self {
```

Comment:
`not_vswhere_latest_but()` I thought that name was strange, sorry!

Reply:
Usage example:

```rust
let vcvars = Vcvars::new()
    .not_vswhere_latest_but(["-version", "[14.0,18.0)"]);
```

It's only a last-resort option anyways, whose use cases are to be replaced with dedicated solutions, when they become known.

Reply:
At first I was thinking of putting only `use_prerelease` then I thought of putting something more specific like `use_vs_prerelease` because basically what differentiated it from is `-prerelease` flag in vswhere with `-latest` will return anything else regular.